### PR TITLE
Lxd Console Logs

### PR DESCRIPTION
**Pull request by Philip-21**
This is pr fixes for #12795

### CODE REVIEW

**Comment by Philip-21** [comment]
from #12795 i made adjustments by adding additional checks for the console log, to indicate `"No new console log messages."`, instead of showing the  `Error: open : no such file or directory` when there are empty log messages

File: lxc/console.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/gorilla/websocket"
 	"github.com/spf13/cobra"
 
-	"github.com/canonical/lxd/client"
+	lxd "github.com/canonical/lxd/client"
```
Comment (tomponline) [inline lxc/console.go:16]: Please can you lose the "lxd" here thanks

Comment (Philip-21) [reply lxc/console.go]: Done

**Review by tomponline — COMMENTED** [review]
Does this actually fix the issue? 
My tests show the error is coming from  error returned from d.GetInstanceConsoleLog() call.

Which is before the changes you've made.

**Comment by Philip-21** [comment]
> Does this actually fix the issue? My tests show the error is coming from error returned from d.GetInstanceConsoleLog() call.
> 
> Which is before the changes you've made.

yes it does, the  stuff variable reads available log messages using the `io.ReadAll()` if logs are empty it returns `No new console log messages.` 
I will have to look at the ` d.GetInstanceConsoleLog() `  to see how console logs are handled

**Comment by Philip-21** [comment]
Hey @tomponline apologies for the late reply, I have ben busy with school activities . 
I will fix surely fix the issue

**Comment by Philip-21** [comment]
Hey @tomponline please what make command do i run to test my changes

**Comment by tomponline** [comment]
The reproducer steps are in https://github.com/canonical/lxd/issues/12795

**Comment by Philip-21** [comment]
> The reproducer steps are in #12795

Thank you.

**Pull request closed without merging**